packages/google_maps_flutter/google_maps_flutter_android/example/integration_test/google_maps_tests.dart

```diff
@@ -21,6 +20,7 @@ const double _kInitialZoomLevel = 5;
 const CameraPosition _kInitialCameraPosition =
     CameraPosition(target: _kInitialMapCenter, zoom: _kInitialZoomLevel);
 const String _kCloudMapId = '000000000000000'; // Dummy map ID.
+const double _floatTolerance = 1e-8;
```

Comment:
Can you add a comment for why this value was chosen?

Reply:
Done